ChangeLog

```diff
@@ -1,3 +1,4 @@
+	* python binding check for invalid torrent parameter
```

Comment:
instead of "invalid torrent parameter", don't you mean "add_torrent_params fields"?